core/src/main/kotlin/me/kcra/takenaka/core/mapping/util/mappings.kt

```diff
@@ -62,19 +62,25 @@ val MappingTreeView.ElementMappingView.hash: String
  * @param mask the modifier mask (you can get that from the [Modifier] class)
  * @return the modifier string, **may end with a space**
  */
-fun Int.formatModifiers(mask: Int = 0): String = buildString {
+fun Int.formatModifiers(mask: Int = 0, interfaceMethod: Boolean = false): String = buildString {
```

Comment:
this method could be completely moved to the `generator-web` module and be used in conjunction with the `ClassType` enum declaring a kind of the parent class, this would open up more formatting possibilities in the future